src/mlpack/methods/ann/layer/base_layer.hpp

```diff
@@ -38,6 +39,13 @@ namespace ann /** Artificial Neural Network. */ {
  *  - IdentityLayer
  *  - ReLULayer
  *  - TanHLayer
+ *  - SoftplusLayer
+ *  - HardSigmoidLayer
+ *  - SwishLayer
+ *  - MishLayer
+ *  - LiSHTLayer
+ *  - GELULayer
+ *  - ELiSHLayer
```

Comment:
Nice Catch! Looks Like most of us missed this. Thanks a lot for this.

Reply:
Thanks @kartikdutt18.